File: 2 Semestre/Tópicos Especiais de Sistemas/Projeto/Academia/Exercicio.cs
class Exercicio
{
    private string _nome;
    public string Nome {
        get => _nome;
    }
    private int _numero;
    public int Numero {
        get => _numero;
    }
    private int _dificuldade;
    public int Dificuldade {
        get => _dificuldade;
    }

    public Exercicio(string _nome, int _numero, int _dificuldade){
        this._nome = _nome;
        this._numero = _numero;
        this._dificuldade = _dificuldade;
    }
}

File: 2 Semestre/Desenvolvimento de Software/Projeto/Projeto-Java/src/entities/Exercicio.java
package entities;
import enums.Dificuldade;
import enums.RegiaoExercicio;

public class Exercicio {
    private String nome;
    private RegiaoExercicio regiao;
    private Dificuldade dificuldade;

    public Exercicio(String nome, RegiaoExercicio regiao, Dificuldade dificuldade) {
        this.nome = nome;
        this.regiao = regiao;
        this.dificuldade = dificuldade;
    }
    public String getNome() {
        return nome;
    }
    public void setNome(String nome) {
        this.nome = nome;
    }
    public RegiaoExercicio getRegiao() {
        return regiao;
    }
    public void setRegiao(RegiaoExercicio regiao) {
        this.regiao = regiao;
    }
    public Dificuldade getDificuldade() {
        return dificuldade;
    }
    public void setDificuldade(Dificuldade dificuldade) {
        this.dificuldade = dificuldade;
    }
}
